doc/user-guide/06-layout-configuration.adoc

```diff
@@ -143,61 +106,60 @@ we mount the +/dev/mapper/backup-backup+ filesystem on
 +/media/backup+, it will get excluded, as the mountpoint is under
 +/media+.
 
-=== Manual excludes
-It seems prudent to prevent the external drives from ever being backed-up or overwritten. The default configuration contains these lines:
+If we mount the filesystem on +/dev/mapper/backup-backup+ on +/backup+,
+Relax-and-Recover will think that it's necessary to recreate the filesystem:
 ----------------------------------
-# Exclude components from being backed up, recreation information is active
-EXCLUDE_BACKUP=()
-
-# Exclude components during component recreation
-# will be added to EXCLUDE_BACKUP (it is not backed up)
-# recreation information gathered, but commented out
-EXCLUDE_RECREATE=()
-
-# Exclude components during the backup restore phase
-# only used to exclude files from the restore.
-EXCLUDE_RESTORE=()
+disk /dev/sdc 1999696297984 msdos
+...
+part /dev/sdc 1073741824000 1048576 primary boot /dev/sdc1
+part /dev/sdc 925953425408 1073742872576 primary lvm /dev/sdc2
+lvmdev /dev/backup /dev/sdc2 cJp4Mt-Vkgv-hVlr-wTMb-0qeA-FX7j-3C60p5 1808502784
+...
+lvmgrp /dev/backup 4096 220764 904249344
+...
+lvmvol /dev/backup backup 12800 104857600 
+lvmvol /dev/backup externaltemp 38400 314572800 
+...
+fs /dev/mapper/backup-backup /backup ext4 uuid=da20354a-dc4c-4bef-817c-1c92894bb002 label= blocksize=4096 reserved_blocks=655360 max_mounts=24 check_interval=180d options=rw
 ----------------------------------
 
-To prevent an inadvertently mounted backup filesystem being added to the restore list, the easiest way is to add the filesystem to the +EXCLUDE_RECREATE+ array.
+=== Manual excludes
+It seems prudent to prevent the external drives from ever being backed-up or overwritten.
```

Comment:
"It seems prudent" sounds smart-alecky.
Simpler and more neutral wording could be like
```
Normally data on external drives should
neither be backed-up during "rear mkbackup"
nor restored during "rear recover".
```

By the way:
I find this whole part confusing.
For me "backup and restore" means backup and restore of data
(files and directories - i.e. what there is in backup.tar.gz)
but here it looks as if disk layout is meant because it then
talks about EXCLUDE_RECREATE which is primarily to exclude
disk layout components during disk layout recreation
and adding them also to EXCLUDE_BACKUP is only an
automated side effect.

Reply:
> For me "backup and restore" means backup and restore of data (files and directories - i.e. what there is in backup.tar.gz) but here it looks as if disk layout is meant because it then talks about EXCLUDE_RECREATE which is primarily to exclude disk layout components during disk layout recreation and adding them also to EXCLUDE_BACKUP is only an automated side effect.

The point of this document is indeed disk layout save and recreation, but I think this phrase is intended to mean that one wants to exclude the drive both from data backup and layout. After all, it is not wrong to point out that `EXCLUDE_RECREATE` does both. I will think how to rephrase it in order to make clear that the main point is exclusion from layout, with exclusion from backup only "by the way".

Reply:
See also https://github.com/rear/rear/issues/3129